packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.controller.js

```diff
@@ -169,4 +169,56 @@ export default class FlavorsListController {
   formatStorage(capacity) {
     return this.$filter('bytes')(capacity, 2, false, 'GB');
   }
+
+  /**
+   * Get the lowest hourly price and lowest monthly price for the flavor
+   * and return distinct prices for both hourly and monthly
+   * @param {*} flavor
+   * @returns Object with lowest hourly / monthly and all distinct hourly / monthly
+   */
+  static getLowestPriceFromFlavor(flavor) {
+    const allHourlyPrices = [];
+    const allMontlyPrices = [];
```

Comment:
**suggestion:** Since we just the need the info whether to display "from", I think we can maintain a boolean rather than maintaining the array.

Reply:
I did that to allow different display if needed. There is 3 different possible cases for each hourly and monthly: 

- case 1 : there is no hourly or monthly price
- case 2 : there is 1 unique hourly / monthly price 
- case 3 : there is multiple different hourly / monthly prices.

For now we use case 1 for houlry to display or not the block, case 2 and 3 to change the translation key depending of the case.

So I think the boolean isn't enought